security-council-rehearsals/templates/r5-guardian-override/.env

```diff
@@ -0,0 +1,9 @@
+# Modify these values according to the Rehearsal instructions
+COUNCIL_SAFE=0xAB23dE0DbE0aedF356af3F815c8B47D88575D82d
+GuardianSafe_ADDRESS=0x91dCf0F534B36218FE7926eac40089eF54006421
+SuperchainConfig_ADDRESS=0xbEDBB0A4d75514c84D569977B14D400c2Af5b36D
```

Comment:
I've been unable to verify either the Proxy or Implementation for this contract.